gramex/handlers/mlhandler.py

```diff
@@ -261,14 +263,18 @@ def _transform(self, data, **kwargs):
         data = self._filterrows(data, **kwargs)
         return data
 
+
     def _predict(self, data=None, score_col=''):
         if data is None:
             data = self._parse_data(False)
         data = self._transform(data, drop_duplicates=False)
         self.model = cache.open(self.model_path, joblib.load)
         try:
             target = data.pop(score_col)
-            return self.model.score(data, target)
+            metric = self.get_argument('_metric')
```

Comment:
If the user does not provide a metric, we need to fall back on the original model of scoring. Please provide for that case also.

Reply:
This line will cause an error if `_metric` is not provided in the URL. The right way to handle this is as follows:

```python
metric = self.get_argument('_metric', False)  # Note that the second argument is returned if the input is not present
if metric:
    scorer = get_scorer(metric)
    return scorer(self.model, data, target)
return self.model.score(data, target)
```